action/lib/parse.js

```diff
@@ -24,18 +50,77 @@ export default function (title, target) {
     return process.exit(0) // soft exit
   }
 
+  let isDev = devDependencyRegEx.test(title);
+  let isProd = dependencyRegEx.test(title);
```

Comment:
I don't think the extra regex here is needed ... it's either a dev dependency or not.

Reply:
I was thinking that we only need to read package.json if we cannot determine whether it is a dev or prod dependency. If we didn't have the 2nd check we would always assume it is a prod dependency in case isDev is false.

What about the following?
1. check package.json if it exists
2. if not, test for devDependency
3. if not, assume production dependency

Reply:
yep that flow makes most sense to me

Reply:
I've implemented the proposal